packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -11,64 +11,46 @@ import { RootDataObject } from "./rootDataObject";
 
 /**
  * Events emitted from {@link IFluidContainer}.
- *
- * @remarks
- *
- * The following is the list of events emitted.
- *
- * ### "connected"
- *
- * The "connected" event is emitted when the `IFluidContainer` completes connecting to the Fluid service.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void;
- * ```
- *
- * ### "dispose"
- *
- * The "dispose" event is emitted when the `IFluidContainer` is disposed, which permanently disables it.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void;
- * ```
- *
- * ### "disconnected"
- *
- * The "disconnected" event is emitted when the `IFluidContainer` becomes disconnected from the Fluid service.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void;
- * ```
- *
- * ### "saved"
- *
- * The "saved" event is emitted when the `IFluidContainer` has local changes acknowledged by the service.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void
- * ```
- *
- * ### "dirty"
- *
- * The "dirty" event is emitted when the `IFluidContainer` has local changes that have not yet
- * been acknowledged by the service.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void
- * ```
  */
 export interface IFluidContainerEvents extends IEvent {
-    (event: "connected" | "dispose" | "disconnected" | "saved" | "dirty", listener: () => void): void;
+    /* eslint-disable @typescript-eslint/unified-signatures */
+    /**
+     * Emitted when the {@link IFluidContainer} completes connecting to the Fluid service.
+     *
+     *
+     * @eventProperty
+     */
+    (event: "connected", listener: () => void): void;
+
+    /**
+     * Emitted when the {@link IFluidContainer} is disposed, which permanently disables it.
+     *
+     * @eventProperty
+     */
+     (event: "dispose", listener: () => void): void;
+
+    /**
+     * Emitted when the {@link IFluidContainer} becomes disconnected from the Fluid service.
+     *
+     * @eventProperty
+     */
+    (event: "disconnected", listener: () => void): void;
+
+    /**
+     * Emitted when the {@link IFluidContainer} has local changes acknowledged by the service.
```

Comment:
Probably good to note that it fires when all have been ack'd, rather than once per ack.

```suggestion
     * Emitted when all of the {@link IFluidContainer}'s local changes have been acknowledged by the service.
```